vars/openshiftEnsureGenericBinaryBuild.groovy

```diff
@@ -0,0 +1,25 @@
+def call () {
+  def genericBuildTemplate = """{
+    "apiVersion": "v1",
+    "kind": "BuildConfig",
+    "metadata": {
+        "name": "generic-component-build"
+    },
+    "spec": {
+        "output": {},
```

Comment:
If this gets built, but not output anywhere, does it just exist in the Docker cache on the node on which it was built (rather than the internal registry). If so, how/when does it get cleared from there?